app/views/employers/_form.html.erb

```diff
@@ -167,25 +165,30 @@
 
     <% if current_user.va_admin? %>
       <div class="row form-group">
-		    <div class="small-12 columns">
-			    <%= f.label :admin_notes, "Admin Notes" %>
-				  <%= f.text_field :admin_notes, class: "form-control" %>
-		    </div>
-	    </div>
+        <div class="small-12 columns">
+          <%= f.label :admin_notes, "Admin Notes" %>
+          <%= f.text_field :admin_notes, class: "form-control" %>
+        </div>
+      </div>
 
       <div class="row form-group">
-  		  <div class="small-12 columns">
-  			  <%= f.label :approved, "Approved" %>
-  			  <%= f.check_box :approved, class: "form-control" %>
-  		  </div>
-  	  </div>
+        <div class="small-12 columns">
+	  <%= f.label :approved, "Approved" %>
+	  <%= f.check_box :approved, class: "form-control" %>
+        </div>
+      </div>
     <% end %>
 
-  	<div class="row">
-		  <div class="small-12 columns">
+    <div class="row">
+      <div class="small-12 columns">
         <div class="actions">
-    		  <%= f.submit class: "button primary" %>
-		    </div>
+	  <% if current_user.va_admin? %>
+	    <%= f.submit class: "button primary", id: "submit-button" %>
+	  <% else %>
+	    <span id="click-button" class="button primary">Update Employer</span>
```

Comment:
I'm still struggling to explain why there are different buttons for admins vs regular users?  Why do we want different validations for these two categories?